### PR TITLE
Scottx611x/integrate trevors observable

### PR DESCRIPTION
**Pull request by scottx611x**
oct-17-2018 16-13-29

### CODE REVIEW

File: refinery/core/views.py [hunk 1]
```diff
@@ -1091,3 +1093,23 @@ def patch(self, request, uuid):
         return Response(
             serializer.errors, status=status.HTTP_400_BAD_REQUEST
         )
+
+
+@api_view(['GET'])
+@staff_member_required
+def site_statistics(request, **kwargs):
+    site_statistics_type = kwargs.get("type")
+    response = HttpResponse()
```
Comment (hackdna) [inline refinery/core/views.py]: Should this be `rest_framework.response.Response` if it's a DRF API view?

File: refinery/core/admin.py [hunk 2]
```diff
@@ -151,7 +151,7 @@ class SiteProfileAdmin(GuardedModelAdmin):
 
 
 class SiteStatisticsAdmin(AdminFieldPopulator):
-    pass
+    change_list_template = "admin/core/sitestatistics/change_list.html"
```
Comment (hackdna) [inline refinery/core/admin.py]: Is folder necessary for a single file?

File: refinery/core/views.py (hunk 1 above)
Comment (scottx611x) [reply refinery/core/views.py]: No, it doesn't have to be. In this case, it's simpler without. To return csv data with a DRF `Response` we would actually have to add a [third party library](https://www.django-rest-framework.org/api-guide/renderers/#csv) to get & use a `CSVRenderer`.

File: refinery/core/admin.py (hunk 2 above)
Comment (scottx611x) [reply refinery/core/admin.py]: I thought it made sense for organizational purposes

File: refinery/core/views.py (hunk 1 above)
Comment (hackdna) [reply refinery/core/views.py]: OK then does it need to be an API view?

Comment (scottx611x) [reply refinery/core/views.py]: The decorator ended up being superfluous

File: refinery/core/urls.py [hunk 3]
```diff
@@ -76,5 +77,6 @@
     url(r'^analyses/(?P<uuid>' + UUID_RE + r')/$',
         AnalysesViewSet.as_view()),
     url(r'^openid_token/$',
-        OpenIDToken.as_view(), name="openid-token")
+        OpenIDToken.as_view(), name="openid-token"),
+    url(r'^sitestatistics/(?P<type>(deltas)|(totals)).csv', site_statistics)
```
Comment (hackdna) [inline refinery/core/urls.py]: If site_statistics is no longer a DRF API view then should it be managed by DRF URL config?

File: refinery/templates/admin/core/sitestatistics/change_list.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{% extends "admin/change_list.html" %}
+{% load i18n admin_urls %}
+
+{% block extrahead %}
+  {{ block.super }}
+  <script type="module">
+    // Load the Observable runtime and inspector.
+    import {Runtime, Inspector} from "https://unpkg.com/@observablehq/notebook-runtime?module";
+
+    import notebook from "https://api.observablehq.com/@scottx611x/refinery-stats.js";
```
Comment (hackdna) [inline refinery/templates/admin/core/sitestatistics/change_list.html]: Shouldn't this live in something like https://api.observablehq.com/@refinery-platofrm instead?

Comment (scottx611x) [reply refinery/templates/admin/core/sitestatistics/change_list.html]: @hackdna I don't believe it should. That would require us to create another github user: `@refinery-platform`. I'd rather have the link tied to a real person than an umbrella user account.

Comment (hackdna) [reply refinery/templates/admin/core/sitestatistics/change_list.html]: So, what would happen if we lost access to https://api.observablehq.com/@scottx611x and needed to make changes to refinery-stats.js?

Comment (scottx611x) [reply refinery/templates/admin/core/sitestatistics/change_list.html]: @hackdna I mean, you could say the same about the refinery-platform github user you are suggesting... Who would have access/administrate that account? Would we be sharing credentials amongst ourselves? If changes really need to be made, the Observable is public and can be forked.

Comment (hackdna) [reply refinery/templates/admin/core/sitestatistics/change_list.html]: Is there no way to provide access to refinery-platform organization?

Comment (scottx611x) [reply refinery/templates/admin/core/sitestatistics/change_list.html]: No, at least not at the moment.

https://beta.observablehq.com/terms-of-service

```
1. Required Information
You must log in with a valid GitHub account, linked to a valid email address in order to complete the signup process. Any other information requested, such as your real name, is optional.
```

**Pull request merged**